src/commcare_cloud/ansible/roles/elasticsearch/templates/systemd/elasticsearch.service.j2

```diff
@@ -9,6 +9,9 @@ Type=simple
 Environment=ES_HOME={{ elasticsearch_home }}
 Environment=CONF_DIR={{ elasticsearch_conf_dir}}
 Environment=ES_PATH_CONF={{ elasticsearch_conf_dir}}
+{% if elasticsearch_version == '5.6.16'%}
```

Comment:
Assuming the same issue exists with newer versions of ES as well, would be nice if this was more flexible. Something along the lines of `elasticsearch_version > 2`

Reply:
I think this should be fixed in ES 6 - See https://github.com/elastic/elasticsearch/issues/25964#issuecomment-318925717

Reply:
That's good. Will we ever run a different _5.x.x_ version? Will this need to be updated if we do?